inputWidget/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Input tag</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="parent">
+        <label class="num" for="num">Enter a number between 1 to 100</label>
+        <input type="number" class="inpElement" value="" />
```

Comment:
label for and input id should match.
 <input type="number" id="num" class="inpElement" value="" />